platforms/shared/configuration/roles/setup/ambassador/tasks/main.yaml

```diff
@@ -7,6 +7,23 @@
 ---
 # We have enabled additional ports on Ambassador for TCP. By default 8443 is exposed.
 # The extra ports are pass as parameter --set ambassador.otherPorts={10010,10020} is taken from network.env.ambassadorPorts
+- name: Set proxy namespace
+  set_fact:
+    proxy_namespace: "{{ network.env.namespace | default('default') }}"
+
+- name: Check if ambassador installed
+  k8s_info:
+    api_version: v1
+    kind: Service
+    name: ambassador
+    kubeconfig: { { kubeconfig_path } }
```

Comment:
Please rename this to ``{{ kubeconfig_path }}`` and similarly to the next line var